src/Utils/TimeConversion.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Provides type or format conversion of time.
  */
-public class Time {
+public class TimeConversion {
```

Comment:
Why do you want to rename this?

Reply:
there is a java.sql.Time class which conflicts with this.